sdk/schemaregistry/schema-registry-avro/test/errors.spec.ts

```diff
@@ -46,21 +46,14 @@ describe("Error scenarios", function () {
       );
     });
     it("a schema with non-avro format", async function (this: Context) {
-      if (isLive) {
-        this.skip();
-      }
-      const schema = await registry.registerSchema({
-        name: "_",
-        definition: "_",
-        format: "NotAvro",
-        groupName: testGroup,
-      });
       await assert.isRejected(
-        serializer.deserializeMessageData({
-          body: Buffer.alloc(1),
-          contentType: `avro/binary+${schema.id}`,
+        registry.registerSchema({
+          name: "_",
+          definition: "_",
+          format: "notavro",
+          groupName: testGroup,
         }),
-        new RegExp(`${schema.id}.*NotAvro.*avro`)
+        /Invalid schema type for PUT request.*notavro/
```

Comment:
This change is unrelated to the fix but I bundled it here too. The service behavior changed such that it catches non-avro schemas at the PUT request time so I unskipped the test and updated it accordingly.